### DISCUSSION TITLE
About text recognition tasks with space data?

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1754] [General]
Hello, how do you mark the space when marking data? How do I modify the profile to start training? After model training, how to display the space on the reasoning result?

**Comment by gaotongxiao** [comment]
For MMOCR 1.0, there's nothing special to including blank space characters in text recognition annotations just as compared to other characters. It's naturally supported by the new data format.

The only thing you need to do is to point `Dictionary` to some space-included dict files. For example:

https://github.com/open-mmlab/mmocr/blob/3aa9572a645ace6fec7e1ab1d1c11038ccbbd908/configs/textrecog/crnn/_base_crnn_mini-vgg.py#L3

You may want to replace `dict_file` with `{{ fileDirname }}/../../../dicts/english_digits_symbols_space.txt`.
